subprojects/docs/src/main/resources/header.html

```diff
@@ -95,92 +95,78 @@
             <li><a href="/userguide/plugin_reference.html">Core Plugins</a></li>
+            <li id="third-party-integration"><a href="/userguide/third_party_integration.html">Gradle &amp; Third-party Tools</a></li>
             <li><a href="https://docs.gradle.com/build-scan-plugin">Build Scan Plugin User Manual</a></li>
         </ul>
 
-        <h3 id="getting-started">Managing Gradle Projects</h3>
+        <h3 id="getting-started">Running Gradle Builds</h3>
         <ul>
-            <li><a href="https://guides.gradle.org/creating-new-gradle-builds/">Creating New Gradle Builds</a></li>
-            <li><a href="https://guides.gradle.org/creating-build-scans/">Inspecting Gradle Builds</a></li>
-            <li><a class="nav-dropdown" data-toggle="collapse" href="#organizing-gradle-projects" aria-expanded="false" aria-controls="organizing-gradle-projects">Organizing Project Structure</a>
-                <ul id="organizing-gradle-projects">
-                    <li><a href="/userguide/organizing_gradle_projects.html">Organizing Projects Best Practices</a></li>
-                    <li><a href="/userguide/directory_layout.html">Directory Layout</a></li>
-                    <li><a href="/userguide/composite_builds.html">Integrating Separate Gradle Builds (Composite Builds)</a></li>
-                    <li><a href="/userguide/intro_multi_project_builds.html">Multi-Project Builds</a></li>
+            <li>Using the Command Line (TBD)</li>
+            <li>Using Gradle From an IDE (TBD)</li>
+            <li><a class="nav-dropdown" data-toggle="collapse" href="#customizing-execution" aria-expanded="false" aria-controls="customizing-execution">Customizing Execution</a>
+                <ul id="customizing-execution">
+                    <li><a href="/userguide/build_environment.html">Configuring the Build Environment</a></li>
+                    <li><a href="/userguide/gradle_daemon.html">Configuring the Gradle Daemon</a></li>
+                    <li><a href="/userguide/init_scripts.html">Creating Initialization Scripts</a></li>
                 </ul>
             </li>
+            <li><a href="/userguide/intro_multi_project_builds.html">Executing Multi-Project Builds</a></li>
+            <li><a href="https://guides.gradle.org/creating-build-scans/">Inspecting Gradle Builds</a></li>
             <li><a class="nav-dropdown" data-toggle="collapse" href="#optimizing-build-performance" aria-expanded="false" aria-controls="optimizing-build-performance">Optimizing Build Times</a>
                 <ul id="optimizing-build-performance">
                     <li><a href="https://guides.gradle.org/performance/">Build Performance Guide</a></li>
                     <li><a href="/userguide/build_cache.html">Enabling and Configuring the Build Cache</a></li>
-                    <li><a href="https://guides.gradle.org/using-build-cache/">Using the Build Cache</a></li>
```

Comment:
This is actually linked from the user manual chapter, so I've removed it from here. Doing so also helps the user, since they don't have to work out what the difference is between the two links on the build cache.

Reply:
👍